src/Magnum/Math/Test/IntersectionTest.cpp

```diff
@@ -204,20 +204,50 @@ void IntersectionTest::aabbFrustum() {
 }
 
 void IntersectionTest::sphereFrustum() {
+    /* Frustum spanning [(-10, -10, -10), (0, 0, 0)] */
     const Frustum frustum{
+        {-1.0f, 0.0f, 0.0f, -10.0f},
         {1.0f, 0.0f, 0.0f, 0.0f},
-        {-1.0f, 0.0f, 0.0f, 10.0f},
+        {0.0f, -0.5f, 0.0f, -5.0f}, /* Intentionally not normalized */
         {0.0f, 1.0f, 0.0f, 0.0f},
-        {0.0f, -1.0f, 0.0f, 10.0f},
         {0.0f, 0.0f, 1.0f, 0.0f},
-        {0.0f, 0.0f, -1.0f, 10.0f}};
```

Comment:
Hmm, so if there's multiple ways how to represent the same plane and the planes can be in any order, maybe the test could be made instanced with testing a few equivalent variants and expecting the same points get always the same result? That could harden it quite nicely.

And that could be done for all `*Frustum` tests -- I see you already updated `planeFrustum()`, so the same instancing change could be done to all.

Reply:
Instanced test for permutating the plane order sounds like it would get out of hand in the test results pretty quickly, I think a loop with iteration description is probably enough?

Re the transformation: Do you mean offsetting/transforming the entire space, frustum and points, sphere's etc by a certain transform?